scripts/check_urls.py

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/python3
+
+import os
+
+def scanDirectory(path):
```

Comment:
as improvement you could use [glob](https://www.geeksforgeeks.org/how-to-use-glob-function-to-find-files-recursively-in-python/) to scan a directory